src/main/webapp/gallery.css

```diff
@@ -22,14 +34,36 @@ body {
 
 .grid-sizer,
 .grid-item {
-  width: 33.333%;
+  width: 25%;
 }
 
 .grid-item {
   float: left;
+  height: auto;
+  padding: 0 !important;
+  margin: 0 !important;
 }
 
 .grid-item img {
   display: block;
   max-width: 100%;
+  width: 100%;
+  height: auto;
+  padding: 0 !important;
+  margin: 0 !important;
+}
+
+/* The overlay effect - lays on top of the container and over the image */
+.overlay {
+  position: absolute;
+  bottom: 0;
+  background: rgb(0, 0, 0);
+  background: rgba(0, 0, 0, 0.5); /* Black see-through */
+  color: #f1f1f1;
```

Comment:
Future reference note: need to change this to variables for a uniform color scheme. I need to see if we can use the color scheme in style.css across css files.